### PR TITLE
Add schedule command info to quickstart

### PR DESCRIPTION
**Pull request by nrb**
Signed-off-by: Nolan Brubaker <nolan@heptio.com>
Co-authored-by: Giancarlo Rubio <gianrubio@gmail.com>

### CODE REVIEW

**Comment by nrb** [comment]
Continuing the work done in #459.

File: docs/quickstart.md [hunk 1]
```diff
@@ -64,6 +64,15 @@ Make sure that you install somewhere in your PATH.
    ark backup create nginx-backup --selector 'backup notin (ignore)'
    ```
 
+1. (Optional) Create backups based on a regular schedule using the `app=nginx` label selector:
+
+    `ark backup create` creates a backup once. `ark schedule create` will create a schedule that creates
+    backups at the specified intervals.
+
+    ```
+    ark schedule create nginx-daily --schedule="@every 1d" --selector app=nginx
```
[the remark this answers is not in the document]
Comment (nrb) [reply docs/quickstart.md]: They should be valid cron expressions. I can update to include that.

**Comment by nrb** [comment]
@metadave Mind taking a look at the update? Does this answer your question?

**Comment by carlisia** [comment]
Lgtm 👍

File: docs/quickstart.md [hunk 2]
```diff
@@ -64,6 +64,23 @@ Make sure that you install somewhere in your PATH.
    ark backup create nginx-backup --selector 'backup notin (ignore)'
    ```
 
+1. (Optional) Create backups based on a regular schedule using the `app=nginx` label selector:
```
Comment (skriss) [inline docs/quickstart.md]: Just some wordsmithing, but I think we can simplify here. WDYT about:

2. (Optional) Set up regularly scheduled backups using a cron expression:
```
ark schedule create nginx-daily --schedule="0 1 * * *" --selector app=nginx
```

   Alternatively....

File: docs/quickstart.md [hunk 3]
```diff
@@ -64,6 +64,23 @@ Make sure that you install somewhere in your PATH.
...
+
+    `ark backup create` creates a backup once. `ark schedule create` will create a schedule that creates
+    backups at the specified intervals. Intervals are specified using standard cron syntax.
+
+    ```
+    ark schedule create nginx-daily --schedule="0 1 * * *" --selector app=nginx
+    ```
+
+    Alternatively, you can use some non-standard shorthand cron expressions
```
Comment (skriss) [inline docs/quickstart.md]: add a `:` at the end of the line

File: docs/quickstart.md (hunk 2 above)
Comment (nrb) [reply docs/quickstart.md]: Makes sense to me, will update.

**Pull request merged**